src/styles/styles.js

```diff
@@ -59,32 +61,41 @@ const webViewStyles = {
         em: {
             fontFamily: fontFamily.GTA,
             fontStyle: 'italic',
+            fontSize: pixelRatio > 1 ? 21 / pixelRatio : 16,
         },
```

Comment:
Can we this DRY

 I see redundant usage of  `fontSize: pixelRatio > 1 ? 21 / pixelRatio : 16`

Reply:
where this redundant?
https://github.com/osama256/App/blob/bc9d377f69cce6b498891c956450deeaa866a0d5/src/styles/styles.js#L61-L65

Reply:
https://github.com/osama256/App/blob/bc9d377f69cce6b498891c956450deeaa866a0d5/src/styles/styles.js#L61-L107

Check the `fontSize` in all the styles.

Reply:
redundant `fontSize` has been removed.